PDSA_System/Server/Controllers/StatusController.cs

```diff
@@ -0,0 +1,46 @@
+using Dapper;
+using Microsoft.AspNetCore.Mvc;
+using PDSA_System.Server.Models;
+
+namespace PDSA_System.Server.Controllers;
+
+[Route("/api/[controller]")]
+[ApiController]
+public class StatusController : ControllerBase
+{
+    private IConfiguration _configuration;
+    public StatusController(IConfiguration configuration)
+    {
+        this._configuration = configuration;
+    }
+
+    // status controller
+    [HttpPost("status")]
```

Comment:
Kunne det vært en idé å endre endpoint fra `/api/status/status` til kun `/api/status`?
```suggestion
    [HttpPost]
```